test.py

```diff
@@ -50,6 +50,10 @@ def test_search(self):
         self.assertTrue(len(result['statuses']) >= 1)
         self.assertEqual(len(result['statuses']), int(result['search_metadata']['count']))
 
+    def test_aggregations(self):
+        result = self.loklak.aggregations('sudheesh001','2015-01-10','2015-10-21',['mentions','hashtags'],10)
+        self.assertEqual(result.status_code,200)
```

Comment:
Hi DengYiping, the result variable (without .status_code) isn't  json data neither, It's that, a 200 response. I have just figured out that the aggregations method returns the direct response, and I needed to add .text to the result/response variable to get what i needed.